docs/src/tool/index.md

```diff
@@ -76,10 +76,9 @@ and each `EXPORT` is an exported Reach.App.
 
 If no `SOURCE` is provided, then `index.rsh` is used.
 
-If no `EXPORT` is provided, then all the exported Reach.Apps will be compiled. If there are no
-Reach.Apps exported, then the program will be compiled as a library, where its exports are available
-to other Reach programs and frontends. The output name of a library is the same as if it exported a Reach.App
-named `default`.
+If no `EXPORT` is provided, then all the exported Reach.Apps will be compiled.
+If there are no Reach.Apps exported, then the program will be compiled as a library, where its exports are available to other Reach programs and frontends.
+The output name of a library is the same as if it exported a Reach.Appnamed `default`.
```

Comment:
The uses of "Reach.App" here should be "`{!rsh} Reach.App`" and this one lost a space